GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -3890,8 +3969,9 @@ subroutine get_GEOS( date_time, force_dtstep, met_path, met_tag,             &
              single_time_in_file = .not. daily_met_files  ! MERRA-2 files are daily files
 
           end if
-          
-          if ( file_exists) then
+
+ 
+          if ( file_exists .and. (.not. is_S2S3_fcst) ) then      ! S2S3 FCST has monthly files 
```

Comment:
@gmao-qliu :  I'm hoping this works, but I'm not 100% sure.  Could this be an issue for S2S3 AODAS forcing (at the very beginning of the AODAS product)?  Maybe `(.not. is_S2S3_fcst)` should be  `(.not. S2S3_file_specs)`??